cmd/issue_show.go

```diff
@@ -54,8 +54,17 @@ var issueShowCmd = &cobra.Command{
 
 		printIssue(issue, rn, renderMarkdown)
 
-		showComments, _ := cmd.Flags().GetBool("comments")
-		if showComments {
+		var noteLevel = NoteLevelNone
+
+		if showComments, _ := cmd.Flags().GetBool("comments"); showComments {
+			noteLevel = NoteLevelComments
+		} else if showActivities, _ := cmd.Flags().GetBool("activities"); showActivities {
+			noteLevel = NoteLevelActivities
+		} else if showFull, _ := cmd.Flags().GetBool("full"); showFull || showComments && showActivities {
+			noteLevel = NoteLevelFull
```

Comment:
@zampierilucas, the second part of this conditional `|| showComments && showActivities` will never be hit: if any of these were set `noteLevel` would already have been set by previous conditionals.

What I suggest is you check things "backwards": get all the flag values before checking, then check for `showComments && showActivities`, then each of one afterwards.

You got what I mean?

Reply:
Yeah, how dumb of me to not notice that :D, I'll send an update